main.rb

```diff
@@ -0,0 +1,8 @@
+require './app'
```

Comment:
- Kindly In order to include classes, modules, etc in other files you have to use require_relative, and when it is some dependency on locating in gem kindly use require eg: require 'date'. so here above I suggest you use require_relative instead of require for app eg: require_relative  './app',  your code looks more professional   :+1: :+1: :+1: